src/app/components/chat/window/output/item/item.styl

```diff
@@ -1,29 +1,94 @@
 @require '~styles/variables'
 @require '~styles-lib/mixins'
-
-$-avatar-size = 32px
+@require '../../variables'
 
 .chat-window-message
 	position: relative
 	margin-bottom: 4px
 	line-height: $font-size-base * 1.25
+	// This is so no height change occurs when the "NEW" indicator line appears
+	border-top-width: 1px
+	border-top-style: solid
+	border-top-color: transparent
 
 	@media $media-xs
 		font-size: $font-size-small
 		line-height: $font-size-small * 1.25
 
 	&-avatar
 		position: absolute
-		left: 0
-		top: 0
-		width: $-avatar-size
+		left: ($left-gutter-size * 0.25)
+		top: 4px
+		width: $avatar-size
 		z-index: 1
 
+		@media $media-xs
+			// On small screens, reduce the left side margin to make more space for the actual messages.
+			left: 4px
+
 		& > img
 			img-circle()
 
 	&-container
-		margin-left: $-avatar-size + 16px
+		margin-left: $avatar-size + $left-gutter-size
+		position: relative
+		transition: background-color 0.1s ease
+		padding-top: 4px
+		padding-left: 4px
+		padding-right: 4px
+		padding-bottom: 2px
+
+		@media $media-xs
+			// On small screens, reduce the left side margin to make more space for the actual messages.
+			margin-left: $avatar-size + 12px
+
+		&:hover
+			background-color: var(--theme-bg-offset)
+			rounded-corners()
+
+			.chat-window-message-small-time
+				opacity: 1
+
+
+	&-small-time
+		position: absolute
+		left: -($avatar-size) - ($left-gutter-size * 0.75)
+		top: 6px
+		font-size: $font-size-tiny
+		theme-prop('color', 'fg-muted')
+		opacity: 0
+		transition: opacity 0.1s ease
+		user-select: none
+
+		// Never show this on small devices. It's probably a phone anyway, and touch can't activate this.
+		@media $media-xs
+			display: none
+
+	&-byline-error
+		cursor: pointer
+		vertical-align: middle
+
+	&-byline-notice
+		vertical-align: middle
+
+	&-queue-error
```

Comment:
Yeah, I'll do this. It was just kind of annoying because of the long class names without them bring properly scoped. I'll do this when I add proper scoping to the rules.